feature-layer-selection/src/main/java/com/esri/arcgisruntime/samples/featurelayerselection/MainActivity.java

```diff
@@ -77,7 +77,6 @@ public boolean onSingleTapConfirmed(MotionEvent e) {
                 Envelope envelope = new Envelope(clickPoint.getX() - mapTolerance, clickPoint.getY() - mapTolerance, clickPoint.getX() + mapTolerance, clickPoint.getY() + mapTolerance, map.getSpatialReference());
                 QueryParameters query = new QueryParameters();
                 query.setGeometry(envelope);
-                query.getOutFields().add("*");
```

Comment:
This change is fine - I find a more fundamental problem that stops the same working correctly, but that's nothing to do with your change, so I will submit a separate issue for that.